src/UI/SlotArea.cpp

```diff
@@ -1114,6 +1116,7 @@ void cSlotAreaAnvil::UpdateResult(cPlayer & a_Player)
 				Output.Empty();
 				SetSlot(2, a_Player, Output);
 				m_ParentWindow.SetProperty(0, 0, a_Player);
+				m_ParentWindow.SendSlot(a_Player, this, 2);
```

Comment:
Output not appearing is what I'd expect from the code above - "clear the Output item, then set it into the output slot". Or am I misunderstanding something?